### PR TITLE
Update properties

### PR DESCRIPTION
**Pull request by gabriel**
```
release update-json --version=1.2 --prop=k1:v1 --prop=k2:v2
{
  "version": "1.2",
  "name": "v1.2",
  "description": "",
  "type": 0,
  "props": [
    {
      "name": "k1",
      "value": "v1"
    },
    {
      "name": "k2",
      "value": "v2"
    }
  ]
}
```

### CODE REVIEW

File: update/update.go [hunk 1]
```diff
@@ -70,13 +71,26 @@ func EncodeJSON(version string, name string, descriptionPath string, src string,
...
 
-		update.Asset = &asset
+		upd.Asset = &asset
 	}
 
-	return json.MarshalIndent(update, "", "  ")
+	if props != nil {
+		uprops := []Property{}
+		for _, p := range props {
+			splitp := strings.SplitN(p, ",", 2)
```
Comment (maxtaco) [inline update/update.go]: why split on comma?

**Comment by maxtaco** [comment]
Rather than:  `--prop=k1:v1`?

**Comment by maxtaco** [comment]
I don't feel super strongly, just seems slightly nicer.

Ok, 👍 on this!

**Comment by zanderz** [comment]
LGTM :thumbsup:
I'll change my go-updater and client PRs to match

**Pull request merged**